usaspending_api/transactions/transaction_delete_journal_helpers.py

```diff
@@ -1,4 +1,4 @@
-import boto3
+import boto3  # noqa
```

Comment:
`# noqa` ? 

Did flake8 flag this as a problem?

Reply:
Yeah, because the typing hint for boto3 is put in quotes since the s3 resource wasn't found.